packages/react-components/react-menu/library/src/components/MenuItem/useMenuItemStyles.styles.ts

```diff
@@ -33,7 +34,7 @@ const useRootBaseStyles = makeResetStyles({
   alignItems: 'start',
   fontSize: tokens.fontSizeBase300,
   cursor: 'pointer',
-  gap: '4px',
+  columnGap: '4px',
```

Comment:
Is it needed?

Reply:
it's cruft from the previous grid implem I'll revert